prisma/migrations/20220531021102_db_setup/migration.sql

```diff
@@ -0,0 +1,67 @@
+-- CreateTable
```

Comment:
Prisma creates and applies these migrations for you based on your schema when you run `npx prisma migrate dev`.  Once you are on prod or staging, you simply need to run `npx prisma migrate deploy`

To create these migrations, you can either 1.) have prisma read the structure of the existing DB or 2.) let the schema.prisma file be the source of truth and generate artifacts from there.